tests/unit/App.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('App component', () => {
     const localVue = createLocalVue()
     localVue.use(VueRouter)
     const router = mockRouter.mock()
-    wrapper = shallowMount(App, { localVue, store, router, vuetify, stubs: { Affix: true } })
+    wrapper = shallowMount(App, { sync: false, localVue, store, router, vuetify, stubs: { Affix: true } })
```

Comment:
Separately from this PR (ie, possibly create a new ticket if this looks significant), please investigate whether the latest version of the test framework / test tools are now async by default. I read somewhere that the tests were initial synchronous, but that breaks some tests (which is why sync: false is needed) and they were changing to async. Maybe we can just go to the next version of their packages now?

Reply:
Very interesting, i will look into this.

Thanks for bringing this up, it appears the Jest devs will be ( if not already have ) moving away from the sync mode as it causes bugs in the tests and doesn't allow simulate how the code runs naturally. So in this case i will update these tests. If we move up dependencies now it may involve a wider refactor ( though i'm not sure we have used sync mode too often ). Either way, ill found out, great topic! 👍

Reply:
So after researching, can confirm that the usage of the `sync` mode in Jest is not recommended. Will be refactoring these unit tests in the current PR, https://github.com/bcgov/bcrs-business-create-ui/pull/49 . 👍